src/components/ViewList.js

```diff
@@ -95,20 +97,42 @@ const ViewList = () => {
       <button onClick={handleClearClick}>Clear</button>
       <ul>
         {currentList.userList.length > 0 ? (
-          filteredList.map((element, index) => (
-            <div key={element.id}>
-              <input
-                type="checkbox"
-                name={element.itemName}
-                id={element.id}
-                value={element.itemName}
-                className="purchased"
-                onChange={handleCheck}
-                checked={element.isPurchased}
-              ></input>
-              <li> {element.itemName} </li>
-            </div>
-          ))
+          filteredList
+            .sort((a, b) => {
```

Comment:
I think during the last weekly meeting Rachel asked about splitting functions. I think the sorting function could be a named function and just referenced here. Since it may be too much for the render?

Other than that, the PR looks great! Great work!

Reply:
This is a great idea. I moved the sorting logic into the useEffect that's setting the filteredList state to avoid sorting at render.